code.rb

```diff
@@ -0,0 +1,56 @@
+module Enumerable
+  def my_each
+    for i in 0...self.length
+      yield (self[i])
+    end
+  end
+
+  def my_each_with_index
+    for i in 0...self.length
+      yield (self[i], i)
```

Comment:
Same with this, let's remove the space.

```suggestion
      yield(self[i], i)
```

Note that space is the reason you having issues with the linter.